app/models/ability.rb

```diff
@@ -11,10 +11,12 @@ def initialize(application = nil)
       can :manage, Move, Move.served_by(application.owner) do |move|
         move.from_location.suppliers.include?(application.owner)
       end
+      can :manage, Journey, supplier_id: application.owner_id
     else
-      # momentarily give all permissions to not break compatibility with
+      # temporarily give all permissions to non-supplier accounts so as not to break compatibility with
```

Comment:
What's the longer term plan here?

Reply:
the longer term plan is to use the external oauth provider and assign roles/rights via that, rather than the internal oauth/doorkeeper we currently have.

Reply:
Okay. I was slightly confused as to my knowledge we only have supplier accounts at present, so I'm not sure what there is that could break? The front end also uses an oauth token tied to a a supplier doesn't it? 🤔

Reply:
the front end uses an api account which is not linked to a supplier and thus has these "super permissions"

Reply:
Interesting, I didn't know that the front end had a different type of auth! I'm glad I asked the question now 🤣